code/src/java/pcgen/cdom/primitive/CompoundAndPrimitive.java

```diff
@@ -58,7 +58,7 @@ public CompoundAndPrimitive(Collection<PrimitiveCollection<T>> pcfCollection)
 				Logging.log(Level.WARNING, "Found duplicate item in "
 						+ pcfCollection);
 			}
-			primCollection.add(PrimitiveCollection.INVALID);
+			primCollection.add(PrimitiveCollection.FIXED.<T>invalid());
```

Comment:
I don;t think I;ve seen that syntax before. Is it valid? If so, do we need to be that clever?